src/lwc/accordionSection/accordionSection.js-meta.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<LightningComponentBundle xmlns="http://soap.sforce.com/2006/04/metadata" fqn="accordionSection">
+    <apiVersion>46.0</apiVersion>
```

Comment:
This needs a label and description. If this is a re-usable utility component, it should indicate that too.